lib/converter.js

```diff
@@ -1,13 +1,14 @@
 "use strict";
 /*global phantom: false*/
 
-var webpage = require("webpage");
+var webpage = require("webpage"),
+system = require("system");
```

Comment:
Bleh, not sure how jshint let you get away with a compound `var` statement. Will fix.